### ISSUE TITLE
E3001: AWS::S3Files resource types not recognized (false positive)

### ISSUE THREAD

**Issue by singledigit** [aws-cloudformation/cfn-lint#4499]
## Description

cfn-lint does not recognize the `AWS::S3Files::*` resource types (e.g., `AWS::S3Files::Copy`), reporting E3001 "Invalid or unsupported Type" errors. These are false positives — the resource types are valid and deploy successfully via CloudFormation.

## Steps to Reproduce

1. Create a CloudFormation template using `AWS::S3Files::Copy` (or other `AWS::S3Files::*` resources)
2. Run `cfn-lint` against the template
3. Observe E3001 errors for each S3Files resource

## Expected Behavior

cfn-lint should recognize `AWS::S3Files::*` as valid CloudFormation resource types without errors.

## Actual Behavior

cfn-lint reports:
```
E3001 Invalid or unsupported Type AWS::S3Files::Copy
```

The linter checks against us-east-1 schemas by default, but `AWS::S3Files` is not in the resource type registry regardless of region — it's a new service that hasn't been added to the CloudFormation Resource Specification yet.

## Environment

- **cfn-lint version:** latest (as of May 2026)
- **Region:** eu-central-1 (deploys successfully)
- **OS:** macOS

## Workaround

Add to `.cfnlintrc`:
```yaml
configure_rules:
  E3001:
    resource_types:
      - AWS::S3Files::*
```

Or use inline ignores in the template.

## Additional Context

- S3 Files is a new AWS service; the CloudFormation Resource Type Specification hasn't been updated to include it yet
- Templates deploy fine to eu-central-1 (and presumably other regions where S3 Files is available)

**Comment by kddejong** [comment]
I have no information about this being a public resource.  Do you have any links?  This may be a privately registered resource which is supported in cfn-lint if you expand your schemas using `-s, --registry-schemas`  [docs](https://github.com/aws-cloudformation/cfn-lint#parameters)

**Comment by singledigit** [comment]
It's a public resource type — here's the official CloudFormation documentation:

- [AWS::S3Files namespace (Template Reference)](https://docs.aws.amazon.com/AWSCloudFormation/latest/TemplateReference/AWS_S3Files.html)
- [AWS::S3Files::FileSystem](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-s3files-filesystem.html)
- [AWS::S3Files::MountTarget](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-s3files-mounttarget.html)

S3 Files is a relatively new service (file system access layered on top of S3). The resource types deploy fine via CloudFormation — they're just not in cfn-lint's registry schemas yet.

**Comment by kddejong** [comment]
My apologies... will be in the release today or you can run `cfn-lint --update-specs` to fix it.

https://github.com/aws-cloudformation/cfn-lint/blob/11231fd395f14bbd6b0073fb80b17a1732e1f89e/src/cfnlint/data/schemas/resources/9b391df7c0bd43ec.json#L225
